### PR TITLE
docs: add dlb entrypoint for contrib

### PR DESCRIPTION
**Pull request by daixiang0**
Signed-off-by: Loong Dai <loong.dai@intel.com>

Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Fixes commit #PR or SHA]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

**Comment by phlax** [comment]
https://status.docker.com/pages/history/533c6539221ae15e3f000031

File: docs/root/api-v3/config/contrib/dlb/dlb.rst [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.. toctree::
```
[the remark this answers is not in the document]
Comment (daixiang0) [reply docs/root/api-v3/config/contrib/dlb/dlb.rst:4]: Updated.

For below link, I do not find how to add title:
https://storage.googleapis.com/envoy-pr/d74536c/docs/intro/arch_overview/security/secpos_requires_trusted_downstream_and_upstream.html?highlight=dlb

File: docs/root/api-v3/config/contrib/dlb/dlb.rst [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Dlb Connection Balance
+======================
```
Comment (phlax) [inline docs/root/api-v3/config/contrib/dlb/dlb.rst:2]:
```suggestion
Dlb Connection Balancer (config)
================================
```

File: docs/root/api-v3/config/contrib/cryptomb/cryptomb.rst [hunk 3]
```diff
@@ -1,3 +1,6 @@
+Cryptomb Private Key Providers
```
Comment (phlax) [inline docs/root/api-v3/config/contrib/cryptomb/cryptomb.rst:1]:
```suggestion
CryptoMB private key roviders
```

same elsewhere re capilatization

**Review by phlax — COMMENTED** [review]
@daixiang0 apart from the capitalization issues in the titles im wondering how we can differentiate the titles - api <> ref docs

eg searching i see this:


namespacing

perhaps we need to add something like `(config)` at the end of all of the titles in the config section - not sure of best way - would be good to fix

File: docs/root/api-v3/config/contrib/cryptomb/cryptomb.rst (hunk 3 above)
Comment (daixiang0) [reply docs/root/api-v3/config/contrib/cryptomb/cryptomb.rst]: Sure, changed.

**Comment by daixiang0** [comment]
I am curious about how is the "api-v3" tagged? If we can control it we can add "config" tag as well.

**Comment by daixiang0** [comment]
@phlax rebased, thanks.

**Review by phlax — APPROVED** [review]
lgtm, thanks @daixiang0

**Pull request merged**